src/target/riscv/riscv-013.c

```
@@ -464,7 +465,11 @@ static dmi_status_t dmi_scan(struct target *target, uint32_t *address_in,
 
 	int retval = jtag_execute_queue();
 	if (retval != ERROR_OK) {
-		LOG_ERROR("dmi_scan failed jtag scan");
+		static int once = 1;
```

Hiding errors makes me nervous. If a target becomes inaccessible I'd prefer a constant stream of errors, just to make sure a user doesn't miss this very important message.